src/sentry/conf/server.py

```diff
@@ -850,8 +850,6 @@ def create_partitioned_queues(name):
     "organizations:internal-catchall": False,
     # Enable inviting members to organizations.
     "organizations:invite-members": True,
-    # Enable selection of members, teams or code owners as email targets for issue alerts.
-    "organizations:issue-alerts-targeting": False,
```

Comment:
@billyvg Will removing this here break the frontend code that's attempting to look at this feature? I assume `organization.features?.includes?.('issue-alerts-targeting');` will just return False if that feature doesn't exist?

Reply:
Yeah it will just be false if it doesn't exist (I'm unsure of the status of the feature though i.e. if it's released, will we now have 0% of users with this)

Reply:
Yeah that is what will happen.

Reply:
Ok, that's fine, the feature is not currently released at all.